rgw/v2/tests/s3_swift/multisite_configs/test_sts_multisite.yaml

```diff
@@ -0,0 +1,34 @@
+# polarion test case id: CEPH-83572938 , CEPH-83574539
+# test scripts : test_sts_using_boto.py , test_sts_using_boto_unexisting_object.py
```

Comment:
Can we use this config file in two of the script mentioned here?

Reply:
done @anrao19